docker/enterprise/docker-compose.enterprise.yml

```diff
@@ -0,0 +1,185 @@
+version: '3.9'
+name: novu
+services:
+  redis:
+    image: 'redis:alpine'
+    container_name: redis
+    restart: unless-stopped
+    logging:
+      driver: 'none'
+  mongodb:
+    image: mongo
+    container_name: mongodb
+    restart: unless-stopped
+    logging:
+      driver: 'json-file'
```

Comment:
how come we are interested in mongo's logs but not redis logs?

Reply:
I would say to debug authentication but otherwise I am unsure